package on2013_11.on2013_11_10_Codeforces_Round__210.A___Levko_and_Array_Recovery;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2013_11/on2013_11_10_Codeforces_Round__210/A___Levko_and_Array_Recovery/A - Levko and Array Recovery.task"))
			Assert.fail();
	}
}
